Dockerfile

```diff
@@ -0,0 +1,30 @@
+# build the server
+FROM golang:alpine as gobuilder
+
+RUN mkdir -p /build/src/github.com/TheFlies/ofriends
+ENV GO_BUILD_ENV=CGO_ENABLED=0 GOOS=linux GOARCH=amd64
+ENV GOPATH=/build
+WORKDIR /build/src/github.com/TheFlies/ofriends
+ADD vendor ./vendor
+ADD internal ./internal
+ADD main.go .
+
+RUN go build -v -o ofriends .
+# build the web
+FROM node:alpine as webbuilder
+RUN mkdir /build
+ADD ./web /build
+WORKDIR /build
+RUN yarn install
```

Comment:
This will be pretty slow as we have to download node modules every time.
Can we just zip node_module and place it somewhere else then inside the webbuilder we just have to unzip it?.